core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -124,7 +177,7 @@ export class LoggedOutUser extends BaseUser {
    * Function to navigate to the home page.
    */
   async navigateToHome(): Promise<void> {
-    await this.goto(homeUrl);
+    await Promise.all([this.page.waitForNavigation(), this.page.goto(homeUrl)]);
```

Comment:
Why do we need this change? is this.goto() not waiting for the navigation? this.goto() should already did the job of waitForNavigation by using `waitUntil: 'networkidle0'`.

Reply:
Ditto for other that didn't specify any waitUntil options inside the waitForNavigation.